boards/x86/up_squared/doc/up_squared.rst

```diff
@@ -13,11 +13,20 @@ along with the following devices:
 
 * Serial Ports in Polling and Interrupt Driven Modes
 
+* I2C
+
+.. note::
+   This board configuration works on all three variants of `UP Squared`_
+   boards containing Intel |reg| Pentium |trade| SoC,
+   Intel |reg| Celeron |trade| SoC, or Intel |reg| Atom |trade| SoC.
+
 .. note::
-   This board configuration only works with the boards containing
-   Intel |reg| Pentium |trade| SoC, or Intel |reg| Celeron |trade| SoC.
-   The board containing Intel |reg| Atom |trade| SoC is not supported
-   by this configuration due to differences in peripherals.
+   This board configuration works only with the default BIOS settings.
+   Enabling/disabling LPSS devices in BIOS (under Advanced -> HAT Configurations)
+   will change the MMIO addresses of these devices, and will prevent
+   the drivers from communicating with these devices. For drivers that support
+   PCI enumeration, enabling CONFIG_PCI and CONFIG_PCI_ENUMERATION will allow
```

Comment:
change to link to the option configuration doc:

    enabling :option:`CONFIG_PCI` and option:`CONFIG_PCI_ENUMERATION` will allow

Reply:
Fixed.